src/components/LicenseRevokeModal/index.jsx

```diff
@@ -47,18 +48,22 @@ class LicenseRevokeModal extends React.Component {
     const {
       user,
       sendLicenseRevoke,
+      fetchSubscriptionDetails,
+      fetchSubscriptionUsers,
+      searchQuery,
     } = this.props;
-
-    const options = {};
-    options.assignments = [{
-      uuid: user.uuid,
-      email: user.emailAddress,
-      licenseStatus: user.licenseStatus,
-    }];
+    const options = { userId: user.userId };
 
     return sendLicenseRevoke(options)
-      .then((response) => {
-        this.props.onSuccess(response);
+      .then(async (response) => {
+        try {
+          await fetchSubscriptionDetails();
+          await fetchSubscriptionUsers({ searchQuery });
+          this.props.onSuccess(response);
+        } catch (error) {
+          // TODO: log error in NewRelic
+          console.log(error); // eslint-disable-line no-console
```

Comment:
I will be filing a ticket related error handling on this page, which will include AC for NewRelic logging.